src/jetstream/docs/swagger.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
This file won't automatically be include when deployed, we'd need to ensure it gets included in container images.

Reply:
It doesn't need to be deployed. There's `docs/docs.go` which is then served as `/swagger/doc.json` by the middleware.
Assuming that we want to use the middleware approach that is.